comp_loinc/build.py

```diff
@@ -1,83 +1,143 @@
+
+    # Example
+    po = PartOntology("./model/schema/part_schema.yaml", "./local_data/part_files")
+    po.generate_ontology()
+    po.write_to_output('./data/output/owl_files/part_ontology.owl')
     """
-    ## Example
-    # po = PartOntology("./model/schema/part_schema.yaml", "./local_data/part_files")
-    # po.generate_ontology()
-    # po.write_to_output('./data/output/owl_files/part_ontology.owl')
     po = PartOntology(schema_file, part_directory)
     po.generate_ontology()
     po.write_to_output(output)
 
 
 @app.command(name='codes')
 def build_codes(
-        schema_file: str,
-        part_directory: str,
-        output: str):
-    """
-    TODO: Refactor this to use the new simplified code_ingest.py
-    :param schema_file:
-    :param part_directory:
-    :param output:
-    :return:
+    schema_file: Path = typer.Argument(default=DEFAULTS['schema_file.codes'], resolve_path=True, exists=True),
+    part_directory: Path = typer.Argument(default=DEFAULTS['part_directory'], resolve_path=True, exists=True),
+    output: Path = typer.Argument(default=DEFAULTS['output.codes'], resolve_path=True, writable=True)
+):
+    """Build codes.
+    :param schema_file: Path to LinkML `.yaml` file that defines data model for LOINC terms, which are identified by
+    LOINC codes.
+    :param part_directory: Path to directory containing TSV files which define the entire LOINC hierarchy of terms and
+    their subcomponent parts.
+    :param output: Path where output will be saved.
+
+    # Example
+    lcc = CodeIngest("./model/schema/code_schema.yaml", "./data/part_files")
+    lcc.write_output_to_file("./data/output/owl_files/code_classes.owl")
     """
-    ## Example
-    # lcc = CodeIngest("./model/schema/code_schema.yaml", "./data/part_files")
-    # lcc.write_output_to_file("./data/output/owl_files/code_classes.owl")
     lcc = CodeIngest(schema_file, part_directory)
     lcc.write_output_to_file(output)
 
 
 @app.command(name='composed')
 def build_composed_classes(
-        schema_file: str,
-        composed_classes_data_file: str,
-        output: str):
-    """
-    todo: this is just calling a linkml owl cli, should be written as code with python
-    :param schema_file:
-    :param code_file:
-    :param output:
-    :return:
+    schema_file: Path = typer.Argument(default=DEFAULTS['schema_file.composed'], resolve_path=True, exists=True),
+    composed_classes_data_file: Path = typer.Argument(
+        default=DEFAULTS['composed_classes_data_file'], resolve_path=True, exists=True),
+    output: Path = typer.Argument(default=DEFAULTS['output.composed'], resolve_path=True, writable=True)
+):
+    """Build composed classes.
+    :param schema_file: Path to LinkML `.yaml` file that defines data model for "grouping classes" of LOINC terms, that 
+    is, classes that group sets of LOINC terms into specific categories.
+    :param composed_classes_data_file: Path to `.yaml` file which lists LOINC composed classes. These are lower-level,
```

Comment:
I'm not sure if I understand grouping classes vs composed classes that well, but here's what I thought / the docstring that I added:
> :param composed_classes_data_file: Path to `.yaml` file which lists LOINC composed classes. These are lower-level, more granular groupings of classes, and their are a greater number of them than the grouping classes in the `schema_file`.